spec/rubocop/cop/rspec/context_wording_spec.rb

```diff
@@ -125,5 +129,72 @@
         RUBY
       end
     end
+
+    context 'when `AllowedPatterns: [/とき$/]`' do
+      let(:cop_config) do
+        { 'Prefixes' => [], 'AllowedPatterns' => [/とき$/] }
+      end
+
+      it 'finds context without `とき` at the ending' do
+        expect_offense(<<-RUBY)
+          context '条件を満たす' do
+                  ^^^^^^^^ Context description should match '(?-mix:とき$)'.
+          end
+        RUBY
+      end
+
+      it 'finds shared_context without `とき` at the ending' do
+        expect_offense(<<-RUBY)
+          shared_context '条件を満たす' do
+                         ^^^^^^^^ Context description should match '(?-mix:とき$)'.
```

Comment:
It seems that when multibyte characters are used here, the caret does not display properly according to the number of characters. This seems to be another issue.

Reply:
We use [unicode-display_width](https://github.com/janlelis/unicode-display_width) which “Determines the monospace display width of a string in Ruby.” Perhaps we don’t use it properly in RuboCop?

> This seems to be another issue.

Yep. I think just adding enough carets to make the specs pass in the correct approach for now 😅